NetKVM/Common/ParaNdis-SM.h

```diff
@@ -6,7 +6,7 @@
 enum class SMNotifications {
     Started,
     Stopped,
-    SupriseRemoved,
+    SurpriseRemoved,
```

Comment:
It's possible this might be a public API and thus not changeable. I'm happy to drop any change(s).